readme.md

```diff
@@ -64,6 +64,12 @@ Values: `'on'`, `'off'`
 
 Determines whether old messages, such as log playbacks, will trigger notifications or not.
 
+### ignore_current_buffer_messages
+
+Default: `'off'`<br>
+Values: `'on'`, `'off'`
+
+Determines whether messages from the current buffer should trigger notifications or not.
```

Comment:
Include the use-case that you mentioned in the PR text.

Reply:
Done